kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -180,4 +188,20 @@ private Object convertLogical(Object kafkaConnectObject,
         LogicalConverterRegistry.getConverter(kafkaConnectSchema.name());
     return converter.convert(kafkaConnectObject);
   }
+
+  private Double convertDouble(Object kafkaConnectObject, Boolean shouldConvertSpecialDouble) {
+    double kafkaConnectDouble = (double)kafkaConnectObject;
+
+    if (shouldConvertSpecialDouble) {
+
+      if (kafkaConnectDouble == Double.POSITIVE_INFINITY) {
```

Comment:
Recommend `if (Double.POSITIVE_INFINITY.equals(kafkaConnectDouble)) {` for NPE safety